### PR TITLE
Updating v1 build to use Azure DevOps.

### LINKED ISSUE #8756: Pipeline Migration: azure-functions-host
Part of #8743

- [x] Migrate pipeline
- [x] Update downstream dependencies
- [x] Cleanup repo
  - [x] Remove obsolete YAML
  - [x] Remove webhook
  - [x] Update badges

### PR DESCRIPTION
**Pull request by kashimiz**
### Issue describing the changes in this PR

addresses #8756 

### Pull request checklist

* [x] My changes **do not** require documentation changes
    * [ ] Otherwise: Documentation issue linked to PR
* [x] My changes **should not** be added to the release notes for the next release
    * [ ] Otherwise: I've added my notes to `release_notes.md`
* [x] My changes **do not** need to be backported to a previous version
    * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [x] My changes **do not** require diagnostic events changes
    * Otherwise: I have added/updated all related diagnostic events and their documentation (Documentation issue linked to PR)
* [x] I have added all required tests (Unit tests, E2E tests)

### Additional information

A couple changes to note:

- Because this branch removes the appveyor.yaml file, the AppVeyor build will not complete as part of the checks. The build that matters is the one on ADO.
- The F# SDK is included in Visual Studio 2019, so the manual PATH addition step has been removed.
- PHP is installed by default on [Microsoft-hosted Windows 2019 build agents](https://github.com/actions/runner-images/blob/main/images/win/Windows2019-Readme.md), so the installation step has been removed.
- Some tests that passed locally and on AppVeyor failed on the ADO build agent. They were disabled in order to proceed, but issues to investigate and fix them have been created. See tracking issue #9003

This PR will be followed by PRs to update the build status badges in branches `dev` and `v3.x`:
- [x] #9039 `dev`
- [x] #9040 `v3.x`

### CODE REVIEW

**Review by liliankasem — APPROVED** [review]

**Comment by liliankasem** [comment]
Definitely a little nervous about disabling the tests but it is for v1 which doesn't get as many changes; would love for the re-enable test issue to be prioritized though

**Review by kshyju — APPROVED** [review]
Echo on Lilian's comment about prioritizing #9003

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+- group: 'Functions Host V1 Testing'
+- name: build_number
+  value: $[ counter('constant', 131400) ]
+- name: includeBuildNumberInVersion
+  value: ${{ 1 }}
+- name: configuration
+  value: Release
+- name: publish_path
+  value: bin
+- name: removeprefix_DOTNET_SKIP_FIRST_TIME_EXPERIENCE
```
Comment (jviau) [inline azure-pipelines.yml]: Curious what `removeprefix_` is for?

**Review by jviau — APPROVED** [review]

**Review by brettsam — APPROVED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (kashimiz) [reply azure-pipelines.yml]: Good question/catch. That's how the environment variable was defined in the AppVeyor build, but comparing it to our other builds, it looks like it shouldn't be there. Removing.

**Pull request merged**